index.html

```diff
@@ -0,0 +1,229 @@
+            <li class="html"><button type="button">html</button></li>
+            <li class="boot"><button type="button">bootstrap</button></li>
+            <li class="ruby"><button type="button">Ruby</button></li>
+          </ul>
+          <div class="project list">          
+            <button class="pr-button" type="button">See Project</button>
+          </div>
+        </div>
+
+        <div class="work4">
+          <h1 class="hash">
+            Profesional Art <br />
+            Printing Data
+          </h1>
+          <p class="pi">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="buttons ul">
+            <li class="html"><button type="button">html</button></li>
+            <li class="boot"><button type="button">bootstrap</button></li>
+            <li class="ruby"><button type="button">Ruby</button></li>
+          </ul>
+          <div class="project list">           
+            <button class="pr-button" type="button">See Project</button>
+          </div>
+        </div>
+
+        <div class="work5">
+          <h1 class="hash">
+            Profesional Art <br />
+            Printing Data
+          </h1>
+          <p class="pi">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="buttons ul">
+            <li class="html"><button type="button">html</button></li>
+            <li class="boot"><button type="button">bootstrap</button></li>
+            <li class="ruby"><button type="button">Ruby</button></li>
+          </ul>
+          <div class="project list">           
+            <button class="pr-button" type="button">See Project</button>
+          </div>
+        </div>
+
+        <div class="work6">
+          <h1 class="hash">
+            Profesional Art <br />
+            Printing Data
+          </h1>
+          <p class="pi">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="buttons ul">
+            <li class="html"><button type="button">html</button></li>
+            <li class="boot"><button type="button">bootstrap</button></li>
+            <li class="ruby"><button type="button">Ruby</button></li>
+          </ul>
+          <div class="project list">          
+            <button class="pr-button" type="button">See Project</button>
+            
+          </div>
+        </div>
+      </section>
+
+      
```

Comment:
Please try to follow the given Figma design. Though your design is not bad, I think it is better to follow the given mobile design.

These sections spacing on your mobile version don't look like the ones in the design. Please try adding the corresponding CSS property to stick to the design as much as possible.

Check the image below to have a better understanding.
![Capture2](https://user-images.githubusercontent.com/75709904/144518941-e8a92b48-310a-4a6d-a78b-b16f0e5af7f9.PNG)

Reply:
Add space.